tests/warp/warp_test.go

```diff
@@ -154,7 +165,53 @@ var _ = ginkgo.Describe("[Warp]", ginkgo.Ordered, func() {
 		log.Info("Creating ethclient for blockchainB", "wsURI", chainBWSURI)
 		chainBWSClient, err = ethclient.Dial(chainBWSURI)
 		gomega.Expect(err).Should(gomega.BeNil())
+	})
+
+	// Deploy the test warp contracts to subnets A & B
+	ginkgo.It("Deploy Test Warp Contracts", ginkgo.Label("Warp", "DeployWarp"), func() {
+		ctx := context.Background()
+
+		auth, err := bind.NewKeyedTransactorWithChainID(fundedKey, chainID)
+		gomega.Expect(err).Should(gomega.BeNil())
 
+		//deploy on subnetA
+		newHeads := make(chan *types.Header, 10)
+		sub, err := chainAWSClient.SubscribeNewHead(ctx, newHeads)
+		gomega.Expect(err).Should(gomega.BeNil())
+
+		addr, tx, _, err := bind.DeployContract(auth, WarpExampleABI, WarpExampleBin, chainAWSClient)
+		gomega.Expect(err).Should(gomega.BeNil())
+		chainANonce++
+		chainAExAddr = addr
+		signed, err := auth.Signer(auth.From, tx)
+		gomega.Expect(err).Should(gomega.BeNil())
+
+		log.Info("Waiting for contract creation on chain A to be accepted")
+		<-newHeads
+		recp, err := chainAWSClient.TransactionReceipt(ctx, signed.Hash())
+		gomega.Expect(err).Should(gomega.BeNil())
+		gomega.Expect(recp.Status).Should(gomega.Equal(types.ReceiptStatusSuccessful)) //make sure status code is 1, contract deployed successfully
+		sub.Unsubscribe()
+
+		//deploy on subnetB
```

Comment:
Since this is the same code to deploy to both chains, could we change this to a for loop over the URIs or respective client values, and deploy the contract once to each so that we reduce the code duplication